src/java/arjdbc/postgresql/PostgreSQLRubyJdbcConnection.java

```diff
@@ -305,46 +340,9 @@ protected void setTimestampParameter(final ThreadContext context,
         final int index, IRubyObject value,
         final IRubyObject attribute, final int type) throws SQLException {
 
-        if ( value instanceof RubyFloat ) {
-            final double doubleValue = ( (RubyFloat) value ).getValue();
-            if ( Double.isInfinite(doubleValue) ) {
-                setTimestampInfinity(statement, index, doubleValue);
-                return;
-            }
-        }
-
-        RubyTime timeValue = toTime(context, value);
-
-        final Timestamp timestamp;
-
-        if (timeValue.getDateTime().getYear() > 0) {
-            timeValue = timeInDefaultTimeZone(context, timeValue);
-            DateTime dateTime = timeValue.getDateTime();
-            timestamp = new Timestamp(dateTime.getMillis());
-
-            if (timeValue.getNSec() > 0) timestamp.setNanos((int) (timestamp.getNanos() + timeValue.getNSec()));
-
-            statement.setTimestamp(index, timestamp, getCalendar(dateTime.getZone()));
-        }
-        else {
-            setTimestampBC(statement, index, timeValue);
```

Comment:
this BC thing has been tricky back and forth. you feel its all good, no regressions in terms of passing tests?

Reply:
yes, tests are passing. In fact I did the normal timestamp handling before the array thing just to make sure.

Reply:
@kares yes, see below. It's fixed with https://github.com/jruby/activerecord-jdbc-adapter/pull/973 . the builds after that look ok

Reply:
no worries. yeah...see #974...that should get 52-stable green...my plan is to iron out a few other things and then start with Rails 6 tests